.editorconfig

```diff
@@ -158,6 +163,307 @@ dotnet_code_quality.api_surface = all
+dotnet_diagnostic.SA1408.severity = none
+# Use trailing comma in multi-line initializers
+dotnet_diagnostic.SA1413.severity = none
+# Tuple types in signatures should have element names
+dotnet_diagnostic.SA1414.severity = none
+# Statement should not be on a single line
+dotnet_diagnostic.SA1501.severity = none
+# Element should not be on a single line
+dotnet_diagnostic.SA1502.severity = none
+# All accessors should be single-line or multi-line
+dotnet_diagnostic.SA1504.severity = none
+# A closing brace should not be preceded by a blank line
+dotnet_diagnostic.SA1508.severity = none
+# Opening braces should not be preceded by blank line
+dotnet_diagnostic.SA1509.severity = none
+# Single-line comments should not be followed by blank line
+dotnet_diagnostic.SA1512.severity = none
+# Closing brace should be followed by blank line
+dotnet_diagnostic.SA1513.severity = none
+# Single-line comment should be preceded by blank line
+dotnet_diagnostic.SA1515.severity = none
+# Code should not contain blank lines at the end of the file
+dotnet_diagnostic.SA1518.severity = none
+# Elements should be documented
+dotnet_diagnostic.SA1600.severity = none
+# Partial elements should be documented
+dotnet_diagnostic.SA1601.severity = none
+# Enumeration items should be documented
+dotnet_diagnostic.SA1602.severity = none
+# The documentation for parameter 'message' is missing
+dotnet_diagnostic.SA1611.severity = none
+# Element parameter documentation should have text
+dotnet_diagnostic.SA1614.severity = none
+# Element return value should be documented
+dotnet_diagnostic.SA1615.severity = none
+# Element return value documentation should have text
+dotnet_diagnostic.SA1616.severity = none
+# The documentation for type parameter is missing
+dotnet_diagnostic.SA1618.severity = none
+# Generic type parameter documentation should have text
+dotnet_diagnostic.SA1622.severity = none
+# Property documentation text
+dotnet_diagnostic.SA1623.severity = none
+# Element documentation should not be copied and pasted
+dotnet_diagnostic.SA1625.severity = none
+# The documentation text within the \'exception\' tag should not be empty
+dotnet_diagnostic.SA1627.severity = none
+# File should have header
+dotnet_diagnostic.SA1633.severity = none
+# Constructor summary documentation should begin with standard text
+dotnet_diagnostic.SA1642.severity = none
+# File name should match first type name
+dotnet_diagnostic.SA1649.severity = none
+
+# Disable some StyleCop rules for test common Program.cs that is linked to test project of template engine
+[src/Tests/Common/Program.cs]
+# Declare types in namespaces
+dotnet_diagnostic.CA1050.severity = none
+# Comments should contain text
+dotnet_diagnostic.SA1120.severity = none
+# Elements should be documented
+dotnet_diagnostic.SA1600.severity = none
+# Partial elements should be documented
+dotnet_diagnostic.SA1601.severity = none
+# File should have header: The file header XML is invalid.
+dotnet_diagnostic.SA1633.severity = none
```

Comment:
The file head with license violates this rule. Disabling in source code like `#pragma warning disable SA1633` still violates this rule because the directive itself is not valid XML.